apps/anoma_node/lib/node/transaction/storage.ex

```diff
@@ -165,21 +173,23 @@ defmodule Anoma.Node.Transaction.Storage do
   def init(args) do
     Process.set_label(__MODULE__)
 
-    keylist =
-      args
-      |> Keyword.validate!([
-        :node_id,
-        uncommitted_height: 0,
-        rocks: false
-      ])
-
-    node_id = keylist[:node_id]
+    args
```

Comment:
I think you forgot to bind `args` here

Also it seems that the rocks option does not matter anymore, does it? Similarly to what you did in logging. Since all of this is done at the configuration level now.

Reply:
No, the validate! will raise if its wrong, so there is no real need to bind it again. But I get the confusion. Ill change it.

Reply:
Yes, but `uncommitted_height: 0` sets the height to 0 if none is provided.